Jenkinsfile

```
@@ -247,7 +247,17 @@ OPENSHIFT_PROMETHEUS_RETENTION_PERIOD=15d
 OPENSHIFT_PROMETHEUS_STORAGE_SIZE=500Gi
 OPENSHIFT_ALERTMANAGER_STORAGE_SIZE=20Gi
               ''')]
-            } else {
+            } else if(env.VARIABLES_LOCATION.indexOf("osp") != -1){
+              build job: 'scale-ci/e2e-benchmarking-multibranch-pipeline/cluster-post-config', parameters: [
+              string(name: 'BUILD_NUMBER', value: BUILD_NUMBER), string(name: 'HOST_NETWORK_CONFIGS', value:'false'),
```

Two vars on the same line here
```suggestion
              string(name: 'BUILD_NUMBER', value: BUILD_NUMBER),
              string(name: 'HOST_NETWORK_CONFIGS', value:'false'),
```